### PR TITLE
Issue #1636 Adding doc links to preflight checks

### PR DESCRIPTION
**Pull request by gunnarmorling**
I found it'd be useful to have the right links in the error messages, making it  bit easier for the user to find that info.

Addresses issue #1636

### CODE REVIEW

**Comment by hferentschik** [comment]
Ok to test

**Comment by hferentschik** [comment]
Welcome @gunnarmorling :-) Mind creating an issue as well?

**Comment by gunnarmorling** [comment]
Yeah, thanks @hferentschik :) Filed and issue and altered the commit message.

**Comment by hferentschik** [comment]
@gunnarmorling, cool thanks. Could you please start the Message "Issue #1636 ..."

**Comment by gunnarmorling** [comment]
Done :)

File: cmd/minishift/cmd/start_preflight.go [hunk 1]
```diff
@@ -107,7 +107,7 @@ func preflightChecksBeforeStartingHost() {
 		checkIsoUrl,
 		"Checking the ISO URL",
 		false, configCmd.WarnCheckIsoUrl.Name,
-		"See the 'Managing Minishift' topic for more information")
+		"See the 'Managing Minishift' topic (https://docs.openshift.org/latest/minishift/using/managing-minishift.html) for more information")
```
Comment (LalatenduMohanty) [inline cmd/minishift/cmd/start_preflight.go]: The topic ` managing-minishift` has changed to https://github.com/minishift/minishift/blob/master/docs/source/using/basic-usage.adoc. Once we release a new version of Minishift (which is today) the docs at https://docs.openshift.org/latest/minishift/index.html

**Comment by gunnarmorling** [comment]
Ok, pushed an update as requested by @LalatenduMohanty.

File: cmd/minishift/cmd/start_preflight.go [hunk 2]
```diff
@@ -107,7 +107,7 @@ func preflightChecksBeforeStartingHost() {
 		checkIsoUrl,
 		"Checking the ISO URL",
 		false, configCmd.WarnCheckIsoUrl.Name,
-		"See the 'Managing Minishift' topic for more information")
+		"See the 'Managing Minishift' topic (https://docs.openshift.org/latest/minishift/using/basic-usage.html) for more information")
```
Comment (robin-owen) [inline cmd/minishift/cmd/start_preflight.go]: The name of the topic itself has also changed from 'Managing Minishift' to 'Basic Usage'. :)

Comment (gunnarmorling) [reply cmd/minishift/cmd/start_preflight.go]: Done.

**Comment by LalatenduMohanty** [comment]
@gbraad Take a look when you are back.

**Pull request merged**